src/webgpu/shader/validation/decl/var.spec.ts

```diff
@@ -527,3 +527,42 @@ g.test('binding_collision_unused_helper')
 
     t.expectCompileResult(true, wgsl);
   });
+
+g.test('address_space_access_mode')
+  .desc('Test that only storage accepts an access mode')
+  .params(u =>
+    u
+      .combine('address_space', ['private', 'storage', 'uniform', 'function', 'workgroup'] as const)
+      .combine('access_mode', ['', ',', ',read', ',write', ',read_write'] as const)
```

Comment:
Can we also cover a trailing comma *after* the access mode? e.g. `var<storage, read_write,>`.

Reply:
Sure

Reply:
Tint also fails `var<storage,read,>` and `var<storage,read_write,>`.